services/gitlab/gitlab-merge-requests.service.js

```diff
@@ -291,7 +292,6 @@ export default class GitlabMergeRequests extends GitLabBase {
     return {
       label: `${labelPrefix}${labelText}merge requests`,
       message,
-      color: mergeRequestCount > 0 ? 'brightgreen' : 'blue',
```

Comment:
We should either set the color to blue here, or we can set the color up in `defaultBadgeData`